### ISSUE TITLE
Add to PyPi?

### ISSUE THREAD

**Issue by ahundt** [jrl-umi3218/Eigen3ToPython#17]
Would you guys be up for adding this to PyPi?

**Comment by haudren** [comment]
I don't think that this should pose any problems. I'll discuss with @gergondet, but in theory it should be quite easy to upload to PyPi.
